src/ChipCarousel/ChipCarousel.scss

```diff
@@ -0,0 +1,24 @@
+@import "variables";
+
+.pgn__chip-carousel {
+  position: relative;
+
+  .pgn__overflow-scroll-overflow-container {
+    --pgn-overflow-scroll-opacity-mask-transparent: rgba(0, 0, 0, 0);
+  }
+}
+
+.pgn__chip-carousel__right-control,
```

Comment:
These selectors are children `.pgn__chip-carousel`?

Reply:
Yes, but if we don't need nesting, it is appropriate to avoid it to have more readable styles.

Reply:
Yeah, I'd say the nesting isn't a hard requirement given we have the convention of BEM class names (mostly) prefixed with `pgn__{componentname}`. Unlikely for collisions to happen, though still possible.